.github/workflows/teardown_application.yaml

```
@@ -0,0 +1,23 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install Flox
+        uses: flox/install-flox-action@v2
+      - name: Dismantle with Pulumi
+        uses: flox/activate-action@v1
+        env:
+          PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+        with:
+          command: mise tasks run infrastructure:down
+ 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)